dash/Dashboard.py

```diff
@@ -144,31 +144,52 @@ def change_tab_focus(self, event):
         if (focus):
             self.center_id = focus #sets center_id to an int or string
             #log.info("Changed focus to {}".format(self.center_id))
+    
+    def get_maneuver(self, id):
+        maneuver_list = {"M_VELKEEP":"VELKEEP", "M_FOLLOW":"FOLLOW", "M_LANESWERVE":"LANE_SWERVE", "M_CUTIN":"CUTIN", "M_STOP":"STOP", "M_REVERSE":"REVERSE"}
```

Comment:
using all caps is a bit aggressive. Either go all lowercase or, preferably, just follow CamelCase like in the actual configs: `VelKeep`, `LaneSwerve`, `CutIn`, etc. so that it matches the code.

Reply:
Additionally, this `maneuver_list` keeps being constructed and destroyed every time the function is called. I'd make it a global variable. Also, it's not a list but a map, so should be `maneuver_map` (or dict).